hbase-shell/src/main/ruby/shell/commands/list_peers.rb

```diff
@@ -38,10 +38,10 @@ def help
       def command
         peers = replication_admin.list_peers
 
-        formatter.header(%w[PEER_ID CLUSTER_KEY ENDPOINT_CLASSNAME
+        table_formatter.start_table({ num_cols: 11, headers: %w[PEER_ID CLUSTER_KEY ENDPOINT_CLASSNAME
```

Comment:
The old didn't bother passing  count of cols.... Would the passed header count ever differ from col count?

Reply:
`num_cols` isn't needed. It is derived from headers and widths by TableFormatter. I just updated the patch to remove `num_cols` from all the places that unnecessarily specified it.